src/inc/quic_datapath.h

```diff
@@ -516,7 +516,7 @@ _IRQL_requires_max_(DISPATCH_LEVEL)
 void
 CxPlatSendDataFreeBuffer(
     _In_ CXPLAT_SEND_DATA* SendData,
-    _In_ QUIC_BUFFER* Buffer
+    _In_ QUIC_BUFFER* Datagram
```

Comment:
Missed one. I still want this called `Buffer`, because it's no longer a datagram in the TCP scenario.